src/model_filereading.cpp

```diff
@@ -96,31 +53,38 @@ void Model::readRadiiAndAtomNumFromFile(std::string& filepath){
       elem_Z[substrings[1]] = std::stoi(substrings[0]);
     }
   }
-  return;
+  // Notify the user if no radius is defined
+  // the program can continue running because the user can manually define radii
+  if (radius_map.size() == 0) {
+    Ctrl::getInstance()->notifyUser("Invalid radii definition file!");
+    Ctrl::getInstance()->notifyUser("Please reload a valid file or define radii manually in the table.");
+  }
 }
 
 bool Model::readAtomsFromFile(std::string& filepath, bool include_hetatm){
 
-  std::vector<Atom> list_of_atoms;
   atom_amounts.clear();
+  raw_atom_coordinates.clear();
 
   if (fileExtension(filepath) == "xyz"){
-    readFileXYZ(list_of_atoms, filepath);
+    readFileXYZ(filepath);
   }
   else if (fileExtension(filepath) == "pdb"){
-    readFilePDB(list_of_atoms, filepath, include_hetatm);
+    readFilePDB(filepath, include_hetatm);
   }
   else { // The browser does not allow other file formats but a user could manually write the path to an invalid file
-    Ctrl::getInstance()->notifyUser("Invalid structure file format!");
+    Ctrl::getInstance()->notifyUser("!!!!!!\nInvalid structure file format!\n!!!!!!");

```

Comment:
Can we please use a sensible number of exclamation marks? By that I mean no more than one at the end of the sentence.

Reply:
Good point. I actually wanted to make it clear that this error message is separate from previous and following lines in the output panel.
How would you feel about dashes/backdashes instead?

Reply:
See my other comment, for for now I think the sentence with an exclamation mark at the end is fine. Only thing we need right now imo, is a way for us to determine that exceptions are correctly found.